docs/content/doc/help/seek-help.en-us.md

```diff
@@ -18,6 +18,13 @@ menu:
 - [Discord](https://discord.gg/NsatcWJ)
 - [Discourse Forum](https://discourse.gitea.io/)
 
+**NOTE:** When asking for support, it may be a good idea to have the following available so that the person helping has all the info they need:
```

Comment:
Again not sure on exact wording, but perhaps also have something like:

"When possible, please try and replicate the issue on [try.gitea.io](https://try.gitea.io) and be sure to include exact steps on how somebody else can recreate the problem as well. There is a much greater chance of solving the issue if it can be reproduced and seen by others."

This is maybe more advice I wish people took when creating issues, but also applies for any type of support

Reply:
Good point, will add.